src/acquisition/fluview/fluview_update.py

```diff
@@ -235,6 +459,16 @@ def main():
     type=int,
     help='issue of the file (e.g. 201740); used iff --file is given'
   )
+  parser.add_argument(
```

Comment:
Are these two flags used anywhere? If not, should they be? Or can they be removed?